tools/check_examples/common.py

```diff
@@ -108,7 +114,7 @@ def load_ignore() -> None:
                     )
 
 
-def ignore(filepath: Path) -> bool:
+def filter_ignore(filepath: Path) -> bool:
```

Comment:
Based on the name I'd expect this to return true for ignored files. Maybe `included` or `not_ignored` or `preserve`?

Reply:
Good idea.  Renamed to filter_not_ignored.